pkgs/by-name/cl/cloudquery/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "cloudquery";
+  version = "6.11.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudquery";
+    repo = "cloudquery";
+    rev = "cli-v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/cli-v${version}";
```

Reply:
Fixed. I think I saw this recommended somewhere before but somehow I didn't use it.